tensorflow_datasets/core/features/video_feature.py

```diff
@@ -130,19 +130,7 @@ def _ffmpeg_decode(self, path_or_fobj):
     ffmpeg_args += self._extra_ffmpeg_args
     ffmpeg_args.append(output_pattern)
     try:
```

Comment:
The `try` / `except` should be moved inside `_run_ffmpeg` too

Reply:
And rather than the `finally:`, you could replace `tempfile.mkdtemp()` by `with tempfile.TemporaryDirectory()` context manager.